src/components/landing/OrbsInBoxes.tsx

```diff
@@ -33,11 +34,11 @@ const orbBox = (margins?: TMargin) =>
     space('space-y-2'),
     margin(margins)
   )
-const highlightedWrapper = (color: Colors) =>
+const highlightedWrapper = (color: Color) =>
   classnames(
     padding('px-2', 'py-1'),
     borderRadius('rounded-full'),
-    backgroundColor(colorToTailwindBg(color)),
+    backgroundColor(colorToTailwindBackground(color)),
```

Comment:
Isn't it already wrapped in `backgroundColor`?

Reply:
This changes the color from 'accent' to 'bg-accent', and then backgroundColor does the type check - should this work differently?